spec/Checker/Refund/MollieOrderRefundCheckerSpec.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace spec\BitBag\SyliusMolliePlugin\Checker\Refund;
+
+use BitBag\SyliusMolliePlugin\Checker\Refund\MollieOrderRefundChecker;
+use BitBag\SyliusMolliePlugin\Checker\Refund\MollieOrderRefundCheckerInterface;
+use Mollie\Api\Resources\Order;
+use PhpSpec\ObjectBehavior;
+
+final class MollieOrderRefundCheckerSpec extends ObjectBehavior
```

Comment:
I would add the scenario with lines [0, 1] and [1, 0] also.

Reply:
Done